sharness/t0020-service-basic-commands.sh

```diff
@@ -31,4 +29,7 @@ test_expect_success "starting a second cluster-service process fails" '
     test_expect_code 1 ipfs-cluster-service --config "test-config"    
 '
 
+test_clean_ipfs
```

Comment:
Hmm it seems that `cleanup ...` does not clean when the tests have failed (even though documentation says it does? https://github.com/chriscool/sharness/blob/master/API.md#final_cleanup).

I understand this fixes that? The reason clean up does not run on that case is probably to allow the user to debug things. I'm fine with this though, since debugging implies running stuff locally and we can then potentially comment this.

Reply:
Yes.  It was tough to see this because as you say the docs are a bit misleading.  But it explained behavior we saw and I'm reasonably confident after looking at sharness source (see link above)

My thoughts exactly.  It makes sense if you want a single failure to stop everything so you can debug, but running this on CI we don't even have a chance to investigate the left over state.